src/app/containers/ArticleFigure/index.jsx

```diff
@@ -73,19 +136,13 @@ const ArticleFigure = ({
           >
             {showCopyright && renderCopyright(copyright)}
           </ImageWithPlaceholder>
-        </ChildWrapper>
-        <ChildWrapper
-          gridColumnStart={1}
-          gridSpan={{
-            default: '6',
-            group3: '5',
-            group4: '5',
-            group5: '10',
-          }}
-        >
-          {renderCaption(captionBlock, type)}
-        </ChildWrapper>
-      </ParentWrapper>
+        </Grid>
+        {hasCaption && (
+          <Grid item columns={captionLayouts[imageOrientation]}>
```

Comment:
This is still rendering when there is no caption - we get a `child is null` error for the `Grid`. Need to investigate futher as to why this is.

Reply:
Resolved in https://github.com/bbc/psammead/pull/2565